### PR TITLE
APPDEV-1229 Fix for scroll issue in friend dashboard

### PR DESCRIPTION
**Pull request by vishal-revdiwala**
- Load next 3 days/week data after scroll
- Rename one class name
- Removed blank lines and dead code

APPDEV-1217 Fix for me-dashboard week date header issue

- Corrected code to get date data
- Updated timezone to UTC

### CODE REVIEW

File: Yona/Yona.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -66,7 +66,7 @@
 		166CB84A1D2C5688008B425D /* MeWeekDetailWeekViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB8491D2C5688008B425D /* MeWeekDetailWeekViewController.swift */; };
 		166CB84D1D2C5E58008B425D /* YonaButtonsTableHeaderView.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB84B1D2C5E58008B425D /* YonaButtonsTableHeaderView.swift */; };
 		166CB84E1D2C5E58008B425D /* YonaButtonsTableHeaderView.xib in Resources */ = {isa = PBXBuildFile; fileRef = 166CB84C1D2C5E58008B425D /* YonaButtonsTableHeaderView.xib */; };
-		166CB8501D2E640A008B425D /* FriendsDayViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB84F1D2E640A008B425D /* FriendsDayViewController.swift */; };
+		166CB8501D2E640A008B425D /* FriendsDashBoardViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 166CB84F1D2E640A008B425D /* FriendsDashBoardViewController.swift */; };
```
Comment (Bert-R) [inline Yona/Yona.xcodeproj/project.pbxproj]: In English, 'dashboard' is a single word, so don't capitalize the `B`

File: Yona/Yona/Friends/FriendsDashBoardViewController.swift [hunk 2]
```diff
@@ -144,64 +120,93 @@ class FriendsDayViewController: MeDashBoardMainViewController {
...
         if selectedTab == .left {
-            performSegue(withIdentifier: R.segue.friendsDayViewController.showFriendsDetailDay, sender: self)
+            performSegue(withIdentifier: R.segue.friendsDashBoardViewController.showFriendsDetailDay, sender: self)
         }
     }
-
     
     // MARK: - Data loaders
-    
     override func loadActivitiesForDay(_ page : Int = 0) {
```
Comment (Bert-R) [inline Yona/Yona/Friends/FriendsDashBoardViewController.swift]: This function is really long. Can't you break it apart?

File: Yona/Yona/Friends/FriendsDashBoardViewController.swift [hunk 3]
```diff
@@ -144,64 +120,93 @@ class FriendsDayViewController: MeDashBoardMainViewController {
...
+                                                     cancelButtonTitle: NSLocalizedString("dashboard.error.button", comment: ""))
+                        alert.show()
+                    })
                 }
             })
         }
     }
-
+    
     override func loadActivitiesForWeek(_ page : Int = 0) {
```
Comment (Bert-R) [inline Yona/Yona/Friends/FriendsDashBoardViewController.swift]: Same here.

File: Yona/Yona.xcodeproj/project.pbxproj (hunk 1 above)
Comment (vishal-revdiwala) [reply Yona/Yona.xcodeproj/project.pbxproj]: Done.

File: Yona/Yona/Friends/FriendsDashBoardViewController.swift (hunk 2 above)
Comment (vishal-revdiwala) [reply Yona/Yona/Friends/FriendsDashBoardViewController.swift]: Yes, I can break into small method.

File: Yona/Yona/Friends/FriendsDashBoardViewController.swift (hunk 3 above)
Comment (vishal-revdiwala) [reply Yona/Yona/Friends/FriendsDashBoardViewController.swift]: Updated code.

**Pull request merged**